modulemd/v2/tests/ModulemdTests/modulestream.py

```diff
@@ -1035,8 +1034,16 @@ def test_depends_on_stream(self):
                 stream.build_depends_on_stream(
                     'platform', 'f28'), False)
 
-            self.assertEqual(stream.depends_on_stream('base', 'f30'), False)
-            self.assertEqual(stream.depends_on_stream('base', 'f30'), False)
```

Comment:
Why are you removing these two tests? Though, the second one should have been `stream.build_depends_on_stream()`, That's a bug in the original code here.